### PR TITLE
Experimental (0.9 Nightly)

### PR DESCRIPTION
**Pull request by K0dikas**
Neovim 0.9 (Nightly):

NEW CHANGES:

    - Noice.nvim,Highly experimental plugin that completely replaces the UI for messages, cmdline and the popupmenu.

        => Added cmdline popup
        => lsp popup messages
        => disabled lsp override (causing bugs all over the time, and it screwed the rust-analyzer's docs)

    - Barbar.nvim, is a tabline plugin with re-orderable, auto-sizing, clickable tabs, icons, nice highlighting, sort-by commands and a magic jump-to-buffer mode. Plus the tab names are made unique when two filenames match.

        => Retangular tabs
        => Move through the buffers, <C-c> to move to the previous buffer and <C-v> to move to the next buffer
        => <C-x> to close the buffer

    - Gitsigns.nvim, integrated it with feline.nvim.

        => statusline integration
        => It shows signs for added, removed, and changed lines, by default, it will show it from the side bar, primarily on the left side

    - feline.nvim, a Lua statusline plugin that prioritizes speed, customizability and minimalism. It's fast and never gets in your way.

        => A Gruvbox statusline theme only (from NV-IDE, but the theme and layout was lost, so I made my own config for it)
        => With lsp diagnostics
        => With icons
        => Gitsigns

    - cellular-automaton.nvim, according to wikipedia, it is a model used in computer science and mathematics. The idea is to model a dynamic system by using a number of cells. Each cell has one of several possible states. With each "turn" or iteration the state of the current cell is determined by two things: its current state, and the states of teh neighbouring cells.

    According to the repository, it is a useless plugin that might help you to cope with stubbornly broken tests or overall lack of sense of life. It lets you execute aethetically pleasing, cellular automaton animations based ont he content of neovim buffer.

    Long story short, it makes animations based on your current neovim buffer.

        => Rain animations
        => Fire animations (looks like fire to me but corret me if I'm wrong)

    - zone.nvim, according to the repository, screensaver plugin for neovim, inspired by emacs-zone.

        => Current configuration is the famous "dvd" logo moving and bouncing. After 60 seconds or 1 minute, it will start movin'.

    - duck.nvim, according to the repository, a duck that waddles between your source codes. If your bored, release the duck, if not, then do it too.

        => Current configuration is set to: duck and cat, these two will start to waddle.

    - The average :Startuptime is now around 80ms.

### CODE REVIEW

**Comment by K0dikas** [comment]
Outdated PR
